packages/core/src/scripts/utils/__tests__/create-symlinks.test.ts

```diff
@@ -0,0 +1,107 @@
+import mock from "mock-fs";
+import createSymlinks from "../create-symlinks";
+import { lstat, realpathSync } from "fs";
+import { tmpdir } from "os";
+import { promisify } from "util";
+
+// We pretend that we're running the command in some project directory, but really we just get an absolute path to a temp directory
+const lstatPromise = promisify(lstat);
+const projectDir = realpathSync(tmpdir());
+
+afterEach(() => {
+  // We need to restore the mock, otherwise it breaks jest
+  //github.com/tschaub/mock-fs#example
+  mock.restore();
+});
+
+beforeEach(() => {
+  // The createSymLinks step is using the CWD to determine the running dir, so we're "mocking" it
+  process.env.CWD = projectDir;
+});
+
+describe("Create Symlink", () => {
+  test("should create symlink if path is valid and exist", async () => {
+    mock({
+      [`${projectDir}/package.json`]: `{
+          "name": "someDep",
+          "dependencies": {
+            "someDep": "./packages/someDep"
+          }
+        }`,
+      [`${projectDir}/packages/someDep`]: {
+        "package.json": `{ name: "someDep" }`,
+      },
+    });
+
+    await createSymlinks();
+
+    const stats = await lstatPromise("./node_modules/someDep");
+    expect(stats.isSymbolicLink()).toBe(true);
+  });
+
+  test("should create symlink if package path is prefixed with file:", async () => {
+    mock({
+      [`${projectDir}/package.json`]: `{
+          "name": "someDep",
+          "dependencies": {
+            "dep-with-file": "file:./packages/dep-with-file"
+          }
+        }`,
+      [`${projectDir}/packages/dep-with-file`]: {
+        "package.json": `{ name: "dep-with-file" }`,
+      },
+    });
+
+    await createSymlinks();
+
+    const stats = await lstatPromise("./node_modules/dep-with-file");
```

Comment:
There is also [`lstatSync`](https://nodejs.org/api/fs.html#fs_fs_lstatsync_path_options) by the way, that I think would be better to use instead I realized, but it's not too important 🙂

Reply:
You don't have to change it, just a comment :)